executor/point_get.go

```diff
@@ -242,21 +245,21 @@ func getColInfoByID(tbl *model.TableInfo, colID int64) *model.ColumnInfo {
 	return nil
 }
 
-// Schema implements the Executor interface.
-func (e *PointGetExecutor) Schema() *expression.Schema {
-	return e.schema
-}
+// // Schema implements the Executor interface.
+// func (e *PointGetExecutor) Schema() *expression.Schema {
+// 	return e.schema
+// }
 
-func (e *PointGetExecutor) retTypes() []*types.FieldType {
-	if e.tps == nil {
-		e.tps = make([]*types.FieldType, e.schema.Len())
-		for i := range e.schema.Columns {
-			e.tps[i] = e.schema.Columns[i].RetType
-		}
-	}
-	return e.tps
-}
+// func (e *PointGetExecutor) retTypes() []*types.FieldType {
+// 	if e.tps == nil {
+// 		e.tps = make([]*types.FieldType, e.schema.Len())
+// 		for i := range e.schema.Columns {
+// 			e.tps[i] = e.schema.Columns[i].RetType
+// 		}
+// 	}
+// 	return e.tps
+// }
 
-func (e *PointGetExecutor) newFirstChunk() *chunk.Chunk {
-	return chunk.New(e.retTypes(), 1, 1)
-}
+// func (e *PointGetExecutor) newFirstChunk() *chunk.Chunk {
+// 	return chunk.New(e.retTypes(), 1, 1)
```

Comment:
General `newFirstChunk` don't use 1, 1 as cap and max chunk size, which wastes memory.

Reply:
Fixed